src/AElf.Kernel.Core/Extensions/BlockExtensions.cs

```diff
@@ -39,12 +39,21 @@ public static void FillTxsMerkleTreeRootInHeader(this IBlock block)
             block.Header.MerkleTreeRootOfTransactions = block.Body.CalculateMerkleTreeRoots();
         }
 
-        public static bool VerifySignature(this IBlock block)
+        public static bool VerifyFormat(this IBlock block)
         {
             if (block.Header.Sig == null || block.Header.P == null)
-            {
                 return false;
-            }
+            if (block.Body.Transactions.Count == 0)
```

Comment:
BlockBody should not contain Transactions @rosona

Reply:
I will remove Body.TransactionList in other PR, Transactions just save transaction ids